components/src/status_im/ui/components/react.cljs

```diff
@@ -64,7 +64,7 @@
 (def image-class (get-class "Image"))
 (def picker-obj (lazy-get-react-property "Picker"))
 (defn picker-class [] (adapt-class (picker-obj)))
-(defn picker-item-class [] (adapt-class (.-Item (picker-obj))))
+(defn picker-item-class [] (adapt-class (.-Item ^js (picker-obj))))
```

Comment:
could you please elaborate why `^js` is needed?

Reply:
it's for improved externs inference https://code.thheller.com/blog/shadow-cljs/2017/11/06/improved-externs-inference.html